### PR TITLE
Drop cloud references from template group names

### PR DESCRIPTION
**Pull request by cnunciato**
These group names are more verbose than what's needed in the Service UI, so this change strips out the cloud names to make them a bit tidier. Additional context [in this thread](https://pulumi.slack.com/archives/C0289AASSG4/p1667500236565769).

This change will have no user-facing impact until the accompanying Service change (https://github.com/pulumi/pulumi-service/pull/11034) is merged.

Part of https://github.com/pulumi/pulumi-service/issues/11031.

### CODE REVIEW

**Comment by scottslowe** [comment]
Quick Q: Will "Web Application" _only_ show up if Kubernetes is selected as the platform?

**Comment by cnunciato** [comment]
I defer to @sean1588, but I'm pretty sure the answer to that is yes (because its `cloud` is `kubernetes`).

**Review by scottslowe — APPROVED** [review]
Approved assuming the behavior is as discussed in the comments (otherwise "Web Application" won't convey to the user enough information).

**Comment by sean1588** [comment]
> I defer to @sean1588, but I'm pretty sure the answer to that is yes (because its cloud is kubernetes).

yes it should. Let me pull this branch down locally and gen the metadata file and test it in the console to verify.

**Comment by sean1588** [comment]
ok, just verified. yes those options only show up when selecting kubernetes. this is how it looks now.
image

**Review by sean1588 — APPROVED** [review]
:shipit:

**Pull request merged**
